src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: lighter;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+}
+
+.data-form {
+  margin: 20px 0;
+}
+
+.input {
+  width: 100%;
+  background: url(images/Search.svg) no-repeat;
+  background-color: #fff;
+  color: #3d4e61;
+  font-weight: 900;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bigInputSearch {
+  height: 70px;
+  line-height: 22px;
+  padding-left: 42px;
+  text-indent: 20px;
+  font-size: 16px;
+  background-size: 19px;
+  background-position-x: 25px;
+  background-position-y: 24px;
+}
+
+.bigInputSearch:hover {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigInputSearch:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.bigInputSearch::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+/* Small Input Search Panel */
+
+.smallInputSearch {
+  height: 42px;
+  line-height: 19px;
+  padding-left: 33px;
+  font-size: 14px;
+  background-size: 11px;
+  background-position-x: 12px;
+  background-position-y: 14px;
+}
+
+.smallInputSearch:hover {
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.smallInputSearch:focus {
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.smallInputSearch::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  line-height: 20px;
+  font-size: 14px;
+  color: #3d4e61;
+}
```

Comment:
Same here for `hover` and `focus` as in `bigInputSearch` class

And, again, you can set styles for states and placeholder for input styles, then just overlap sizes for `smallInputSearch` class, there is no need to duplicate all styles